Client/mods/deathmatch/logic/CPacketHandler.cpp

```diff
@@ -3078,6 +3079,9 @@ void CPacketHandler::Packet_EntityAdd(NetBitStreamInterface& bitStream)
                         if (bitStream.Read(&health))
                             pObject->SetHealth(health.data.fValue);
 
+                        if (bitStream.ReadBit())
+                            pObject->Break();
```

Comment:
This is going to fail if either the server is running an older version (and doesn't send this bit) or when the client is older (and the server sends a bit here). You have to check if the bitstream supports this.